client/src/PublicRequest.ts

```diff
@@ -189,6 +190,40 @@ export type SignTransactionRequest
     | SignTransactionRequestCheckout
     | SignTransactionRequestCashlink;
 
+export type EncryptionKeyParams = {
+    kdf: string,
+    iterations: number,
+    keySize: number,
+};
+
+export type MultisigConfig = {
+    publicKeys: Uint8Array[],
+    signers: Array<{
+        publicKey: Uint8Array,
+        commitments: Uint8Array[],
+    }>,
+    secrets: Uint8Array[] | {
+        encrypted: Uint8Array[],
+        keyParams: EncryptionKeyParams,
+    },
+    userName?: string,
+};
+
+export type SignMultisigTransactionRequestCommon = Transform<SignTransactionRequestCommon, 'keyLabel' | 'senderLabel', {
+    keyLabel: string, // Not optional
+    senderLabel: string, // Not optional
+}> & {
+    multisigConfig: MultisigConfig,
+};
```

Comment:
Could also be
```suggestion
export type SignMultisigTransactionRequestCommon = Transform<
    SignTransactionRequestCommon,
    'keyLabel' | 'senderLabel',
    Required<Pick<SignTransactionRequestCommon, 'keyLabel' | 'senderLabel'>>
> & {
    multisigConfig: MultisigConfig,
};
```
This ensures that there are no discrepancies if the original types of `keyLabel` or `senderLabel` were to change in `SignTransactionRequestCommon`, or if they were to be removed there. Obviously, in this case this benefit is of a rather theoretical nature. I just wanted to mention this alternative.

Reply:
Resolved in #524